### PR TITLE
[PAXLOGGING-263] Log4j should be configurable using defaults file.

### PR DESCRIPTION
**Pull request by hema-1046**

### CODE REVIEW

File: pax-logging-log4j2/src/main/java/org/ops4j/pax/logging/log4j2/internal/PaxLoggingServiceImpl.java [hunk 1]
```diff
@@ -66,6 +69,7 @@
         implements PaxLoggingService, LogService, ManagedService, ServiceFactory {
 
     private static final String LOGGER_CONTEXT_NAME = "pax-logging";
+    private static final String LOG4J2_CONFIG_DEFAULTS_KEY = "org.ops4j.pax.logging.log4j2.defaults.file";
```
Comment (jbonofre) [inline pax-logging-log4j2/src/main/java/org/ops4j/pax/logging/log4j2/internal/PaxLoggingServiceImpl.java]: What's the point compared to `org.ops4j.pax.logging` configuration ?

Comment (jbonofre) [reply pax-logging-log4j2/src/main/java/org/ops4j/pax/logging/log4j2/internal/PaxLoggingServiceImpl.java]: You want an additional file to separator "core/default" appenders/loggers from yours ?

Comment (hema-1046) [reply pax-logging-log4j2/src/main/java/org/ops4j/pax/logging/log4j2/internal/PaxLoggingServiceImpl.java]: yup, exactly.

Comment (jbonofre) [reply pax-logging-log4j2/src/main/java/org/ops4j/pax/logging/log4j2/internal/PaxLoggingServiceImpl.java]: OK, understood.

**Pull request merged**